Packs/AWS-IAMIdentityCenter/ReleaseNotes/1_0_2.md

```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+
+##### AWS - IAM Identity Center - TEST
+
+- %%UPDATE_RN%%
```

Comment:
Please proper release notes mention all the commands you added and edited. Also remove this line.